src/ontology/mondo-edit.obo

```diff
@@ -320636,7 +320638,7 @@ is_a: MONDO:0024573 {source="DOID:0110315", source="MESH:C566044", source="MONDO
 is_a: MONDO:0100494 {source="https://orcid.org/0000-0001-9310-0163"} ! autosomal dominant titinopathy
 intersection_of: MONDO:0005045 ! hypertrophic cardiomyopathy
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12403 ! TTN
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12403 {source="MONDO:mim2gene_medgen", source="OMIM:613765"} ! TTN
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12403 {source="MONDO:mim2gene_medgen"} ! TTN
```

Comment:
This gene association is marked as provisional, phenotype name starts with `?`, https://omim.org/entry/613765 and therefore does not meet the criteria to be maintained by the omim ingest pipeline. Please confirm if this needs to be maintained and added to the "protected" file.

Reply:
This should be maintained and added to the "protected" file